code/modules/projectiles/gun.dm

```diff
@@ -15,29 +15,35 @@
 	throw_range = 5
 	force = 5.0
 	origin_tech = "combat=1"
-	attack_verb = list("struck", "hit", "bashed")
+	attack_verb = list("struck", "hit", "bashed", "slapped")

```

Comment:
Slapped, really? I'd say slapping if more something you do with your bare hands.